packages/react-layouts/src/PageContent/PageContent.md

```diff
@@ -0,0 +1,37 @@
+## Overview
+
+`PageContent` is a simple content wrapper that automatically fades-in content on load, and hides content if the `loading` prop is set to true.
+
+## Use
+
+The following contrived example loads and un-loads page content continuously so you can see the sweet transition animation!
+
+```jsx
+import { Heading, Text } from '@puppet/react-components';
+
+const PerpetuallyLoadingExample = () => {
+  const [loading, setLoading] = React.useState(false);
+
+  React.useEffect(() => {
+    const timeout = setTimeout(() => setLoading(!loading), 2000);
+
+
+    return () => {
+      clearTimeout(timeout);
+    };
+  }, [loading]);
+
+  return (
+    <PageContent
+      loading={loading}
+      type="secondary"
+      style={{ height:500, display: 'flex', textAlign: 'center' }}
+    >
+      <Heading>Sample</Heading>
+      <Text>Content</Text>
+    </PageContent>
+  )
+}
+
+<PerpetuallyLoadingExample />
```

Comment:
Just an FYI, Styleguidist is actually smart enough to not require that you wrap this in a component even with hooks.